website/static/js/fangorn.js

```diff
@@ -2541,8 +2543,14 @@ function _dropLogic(event, items, folder) {
             });
         } else {
             toMove.ready.forEach(function(item) {
-                doItemOp.call(tb, copyMode === 'move' ? OPERATIONS.MOVE : OPERATIONS.COPY, folder, item, undefined, 'replace');
             });
+            var moveInterval = setInterval(function() {
```

Comment:
I am wondering if this is interfering with the modal that lets you make decisions when there are duplicate files, for example.  Here, I drag a list of files, one of which has a duplicate in the destination.  The modal warning me of this just flashes briefly and then closes.  The one file I'm asked to confirm after moving doesn't make sense.


![animated gif-downsized_large 6](https://user-images.githubusercontent.com/9755598/53828062-3caef080-3f42-11e9-8903-3af46b9469ff.gif)